hello-world/index.js

```diff
@@ -0,0 +1,13 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var multer = require('multer');
+var upload = multer();
```

Comment:
@psinha20 You're using the import 'upload'? Is it necessary?

Reply:
@psinha20 Use callback function in 'app.listen(8000)'  and print message like (It makes more clear about server is starting)

```
app.listen(8000,  function() {
  console.log('server is starting on port 8000 ')
});
```